### ISSUE TITLE
Velux Theme Homepage

### ISSUE THREAD

**Issue by tussinglady** [godaddy/wp-velux-theme#51]
I was having the same issues as #49 and so I followed the instructions of downloading the new version of velux and installing it. But now my homepage is different. Instead of the awesome giant picture with "Make a Bold Statement", it's the cropped version of the photo with "HOME" written. I much preferred the larger picture. How do I keep that same Home page?

**Comment by EvanHerman** [comment]
Hi @tussinglady ,

Do you have a link to your website? The homepage should have persisted between the two versions of the theme, as nothing major has changed within the theme itself.

What version of Primer and Velux do you have installed? I'd be happy to do some testing on my end.

Best,
Evan

**Comment by tussinglady** [comment]
Thanks for being so prompt Evan!


Here is the link to the website, hopeforbukwo.org


So I went to theme details for Velux and it says it's my current theme, active, and 1.1.0 and then I looked at Primer it's not active and it's 1.8.0


I apologize in advance, I'm fairly novice to WordPress.


Illyssa Tussing

[1499372093745_Full]

creativetechaz.com<http://www.creativetechaz.com>

[1499372431544_FB2.jpg]<http://www.facebook.com/creativetechaz>[1499372471778_Insta2.jpg]<http://www.instagram.com/creativetechaz>

________________________________

**Comment by tussinglady** [comment]
@EvanHerman 
Any ideas on this yet? I deleted the zip file version of the theme and installed the one listed on WordPress and the home image is correct but of course there's the issue with the drop down menus.

**Comment by EvanHerman** [comment]
Hi @tussinglady ,

The menu issue inside of Velux was [recently reported](https://github.com/godaddy/wp-velux-theme/issues/49) and has been merged into our develop branch. We are waiting on releasing the next version, but in the meantime the latest release can be downloaded from here on Github.

[Velux (Development Branch)](https://github.com/godaddy/wp-velux-theme) - [Download Link](https://github.com/godaddy/wp-velux-theme/archive/develop.zip)

Once you have the .zip you'll want to just change the name of it to *velux* before uploading it to your WordPress site. You should find that the menu issue is resolved on that branch. when we push out the next release of Velux (most likely v1.1.1) the issue will be resolved on the master branch as well.

Let me know if that helps.

Best,
Evan

**Comment by tussinglady** [comment]
@EvanHerman 
It still does the same thing? It makes the main image on the home page shrink. If you go to my website: hopeforbukwo.org you can see that. What is doing that?

**Comment by EvanHerman** [comment]
I can't access the site, I see a "This site can't be reached" notice.

**Comment by EvanHerman** [comment]
The reason the hero is not expanded fully is because you have no hero widget and the hero collapses. If you want to make the hero larger, you can add a hero widget inside of the 'Appearance > Widgets' menu.

Example:
<img src="https://cldup.com/XRpCXd_Nua.png" width="50%" />

Otherwise, you will need to add custom CSS into the 'Additional CSS' section of the customizer. (Appearance > Customize > Additional CSS)

<img src="https://cldup.com/9d-ZpcMyNb.png" width="50%" />

I don't know your ultimate goal here, but a few options of CSS to expand the header, which you can add into the 'Additional CSS' section are:

```css
body.home .page-title-container {
   margin-bottom: 2em;
}
```

or

```css
body.home #masthead {
   height: 100vh;
}
```

or

```css
body.home #masthead {
   min-height: 500px;
}

.home .page-title-container {
   margin-top: 5%;
}
```

There are a few ways you can go about tweaking the header size, but feel free to try any of the approaches above and tweak to your satisfaction. I think the last one is probably what you are after, since it will increase the header height and tweak the header title top margin - but keep in mind that you will also need to tweak the styles for the mobile breakpoints.


Evan

**Comment by tussinglady** [comment]
It's totally the hero widget! Awesome @EvanHerman thank you SO much.

**Comment by EvanHerman** [comment]
No problem at all - sorry that it took so much back and forth to get to the bottom of that!

I hope you have a wonderful rest of your day. If you run into any other issues moving forward don't hesitate to reach back out.

Evan
